actk/bin/all.py

```diff
@@ -99,17 +102,27 @@ def run(
                 # Create cluster
                 log.info("Creating SLURMCluster")
                 cluster = SLURMCluster(
-                    cores=4,
-                    memory="20GB",
+                    cores=1,
+                    memory="6GB",
                     queue="aics_cpu_general",
                     walltime="10:00:00",
                     local_directory=str(log_dir),
                     log_directory=str(log_dir),
                 )
-                log.info("Created SLURMCluster")
 
-                # Scale cluster
-                cluster.scale(60)
+                # Spawn workers
+                cluster.scale(100)
+
+                # Wait for workers
+                # In a dream situtation we could call
+                # client = Client(cluster)
+                # client.wait_for_workers(100)
+                # But that doesn't work
+                # instead...
+                log.info("Waiting for workers to start...")
+                time.sleep(120)
```

Comment:
how do you know 120 is enough?  is there a more robust way to check?  what happens if you don't sleep here?

Reply:
There is literally a function called `client.wait_for_workers(some_integer)` but because our cluster never gives us all the workers we request I found this to be pretty good.

Nothing really happens if I don't include this. I just like it this way because the log and dask monitor make more sense. You will have to wait for the workers to start either way this just _feels_ better to look at the dask monitor and all the workers have been created.